### PR TITLE
version update to 0.2.13

### PR DESCRIPTION
**Pull request by tomzhang**

### CODE REVIEW

File: config/haproxy_template.cfg [hunk 1]
```diff
@@ -53,40 +57,57 @@ frontend http-in
...
-{{ range $index, $app := .Apps }} {{ if $app.Env.BAMBOO_TCP_PORT }}
-listen {{ $app.EscapedId }}-cluster-tcp :{{ $app.Env.BAMBOO_TCP_PORT }}
-        mode tcp
-        option tcplog
-        balance roundrobin
-        {{ range $page, $task := .Tasks }}
-        server {{ $app.EscapedId}}-{{ $task.Host }}-{{ $task.Port }} {{ $task.Host }}:{{ $task.Port }} {{ if $app.HealthCheckPath }} check inter 30000 {{ end }} {{ end }}
-{{ end }}
+frontend https-in
+        bind *:443 ssl crt /etc/ssl/dataman.io.pem
```
Comment (bluepeppers) [inline config/haproxy_template.cfg]: It's probably not appropriate to have the ssl cert paths of a specific organisation in our default template

File: configuration/configuration.go [hunk 2]
```diff
@@ -36,27 +38,56 @@ type Configuration struct {
...
+func fromEnv(conf *Configuration) {
+	setValueFromEnv(&conf.Marathon.Endpoint, "MARATHON_ENDPOINT")
 	setValueFromEnv(&conf.Bamboo.Endpoint, "BAMBOO_ENDPOINT")
 	setValueFromEnv(&conf.Bamboo.Zookeeper.Host, "BAMBOO_ZK_HOST")
 	setValueFromEnv(&conf.Bamboo.Zookeeper.Path, "BAMBOO_ZK_PATH")
-
 	setValueFromEnv(&conf.HAProxy.TemplatePath, "HAPROXY_TEMPLATE_PATH")
 	setValueFromEnv(&conf.HAProxy.OutputPath, "HAPROXY_OUTPUT_PATH")
 	setValueFromEnv(&conf.HAProxy.ReloadCommand, "HAPROXY_RELOAD_CMD")
+	setBoolValueFromEnv(&conf.StatsD.Enabled, "STATSD_ENABLED")
```
Comment (bluepeppers) [inline configuration/configuration.go]: StatsD is UDP based, and very lightweight, so I question the need to disable it entirely; just point it at localhost if you don't have a stastd server.

File: main/bamboo/bamboo.go [hunk 3]
```diff
@@ -72,14 +114,30 @@ func main() {
 	eventBus.Register(handlers.MarathonEventHandler)
 	eventBus.Register(handlers.ServiceEventHandler)
 	eventBus.Publish(event_bus.MarathonEvent{EventType: "bamboo_startup", Timestamp: time.Now().Format(time.RFC3339)})
+<<<<<<< HEAD
```
Comment (bluepeppers) [inline main/bamboo/bamboo.go]: This has not been merged correctly

File: services/event_bus/events.go [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+func generatePlaintext(m map[string]interface{}) string {
+	keys := make([]string, 0)
+	for k, _ := range m {
+		if eventKeys[k] {
+			keys = append(keys, k)
+		}
+	}
+<<<<<<< HEAD
```
Comment (bluepeppers) [inline services/event_bus/events.go]: This has not been merged correctly.

**Comment by bluepeppers** [comment]
I think it would be easier to review this if it were broken up into individual pull requests for each feature. There are some nice features here, but it is hard to see them at the moment.

**Comment by activars** [comment]
Can you rebase the commits? It will produce cleaner diffs for review.
Thank you.

File: config/haproxy_template.cfg (hunk 1 above)
Comment (tomzhang) [inline config/haproxy_template.cfg]: that's true , just forget to removed it .

**Pull request closed without merging**